[PE-1327] Increasing overall create/delete times

kevinkirkup opened the pull request:
Still seeing timeouts when deleting Google connections.
Increasing overall timeouts for everyone.

Try75 commented on website/docs/r/site_vpn_connection.html.md:
```diff
@@ -3,7 +3,7 @@ layout: "pureport"
 sidebar_current: "docs-pureport-resource-site_vpn_connection"
 description: |-
-  Manages a Pureport AWS Direct Connect Connection.
+  Manages a Pureport Site VPN Connect Connection.
```
Can we drop Connect here?

kevinkirkup replied:
done.

Commit:
Increasing overall create/delete times

Still seeing timeouts when deleting Google connections.
Increasing overall timeouts for everyone.
